react-app/src/components/Banner/latestUpdates.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useState } from "react";
+
+import { cn } from "@/utils";
+
+interface LatestUpdatesProps {
+  className?: string;
+}
+
+export const LatestUpdates: FC<LatestUpdatesProps> = ({ className }) => {
+  const [showMore, setShowMore] = useState(false);
+
+  return (
+    <div className={cn("m-auto w-full max-w-screen-xl min-h-[228px]", className)}>
+      <div className="border-2 rounded-b-sm p-4 md:p-6rounded-[3px] border-[#DFE1E2] bg-[var(--Background-Base-base-lightest,rgba(240,240,240,1))] overflow-hidden">
+        <h2 className="text-2xl font-bold font-serif pb-2">Latest Updates</h2>
+        <ul className="list-none pl-1">
+          <li className="pb-8 relative pl-4 before:content-['•'] before:absolute before:left-0 before:text-sm before:top-1.5 before:leading-none">
+            11-12-24
+            <span className="font-bold text-[#0071bc]">
+              {" "}
+              MMDL SPA forms available in OneMAC:
+            </span>{" "}
+            Medicaid Alternative Benefit Plan, Premium and Cost Sharing, and CHIP Eligibility SPA
+            templates and implementation guides are now available in OneMAC. New submissions for
+            these SPA types are submitted through the OneMAC system effective [date].
```

Comment:
I know we're in an unclear state for the updates/notifications but should we be pulling the notifs from the getNotifs api or do we want to hardcode them for now @jdinh8124?

Reply:
@daniel-belcher 
Maybe the API just so we can test functionality and use it in practice